core/Enclave/enclave_hsm.cpp

```diff
@@ -489,30 +655,109 @@ sgx_status_t enclave_rsa_decrypt(const uint8_t *cmk_blob, size_t cmk_blob_size,
 {
     sgx_status_t ret = SGX_ERROR_UNEXPECTED;
 
-    rsa_key_data_t *temp = (rsa_key_data_t*)cmk_blob;
-
-    BIO *bio = BIO_new_mem_buf(temp->rsa_private_key, -1);
-    RSA *rsa = PEM_read_bio_RSAPrivateKey(bio, NULL, NULL, g_domain_key);
-    
+    if (keyspec == NULL || padding_mode == NULL)
+        return SGX_ERROR_INVALID_PARAMETER;
 
-    if(rsa == NULL)
+    uint32_t sealed_rsa_len;
```

Comment:
what's the meaning for "sealed_rsa_len"?

Reply:
it will be changed